data/transactions/logic/TEAL_opcodes.md

```diff
@@ -425,6 +425,8 @@ Overflow is an error condition which halts execution and fails the transaction.
 | 53 | GlobalNumByteSlice | uint64 | Number of global state byteslices in ApplicationCall. LogicSigVersion >= 3. |
 | 54 | LocalNumUint | uint64 | Number of local state integers in ApplicationCall. LogicSigVersion >= 3. |
 | 55 | LocalNumByteSlice | uint64 | Number of local state byteslices in ApplicationCall. LogicSigVersion >= 3. |
+| 56 | AppProgramExtraPages | uint64 |  |
+| 57 | CreatableID | uint64 | The assigned creatable ID for an asset or application creation transaction, or 0 if the transaction does not create anything. LogicSigVersion >= 4. |
```

Comment:
Also, I think we need to use "CreatedID" to be more clear that this doesn't just tell you the id of the asa or app the txn is working on, it's the one that's created in this transaction.

Reply:
I agree that CreatedID is more clear, but I noticed that creatable ID is already widely used within the codebase so I was hesitant to go against the existing precedent. Let me know if you still think it's worth changing though, since it's an easy change and I'm not too tied to one or the other.

Reply:
Let me look around at the uses.  I think it's ok to make it Created if Creatable is used more generically - any id for a creatable resource.  But if we are explicitly using that phrase only around an actual id created in a given transaction, then we should stick with it.  That is, I agree that this "a" CreatableID (in some sense that's its type), but this particular one is the CreatedID of the transaction.